db/db.js

```diff
@@ -365,6 +383,17 @@ async function isOverdoseSuspected(xethru, session, location) {
     let condition2 = 1 * (session.still_counter > still_threshold); //seconds
     let condition3 = 1 * (sessionDuration > sessionDuration_threshold);
     let condition4 = 1 * (0);
+
+    let alertReason='NotSet'
+
+    if(condition2 == 1){
+      alertReason = 'Duration';
+    }
+
+    if( (condition1==1) || (condition3==1) ){
```

Comment:
Same here. It would be more consistent with our formatting to have spaces around the `==`s

Reply:
Noting this, and will add a linter to the list of things in the next PR.

Reply:
Cool. This is the ESLint configuration that we're currently using in Buttons and the shared library:
https://github.com/bravetechnologycoop/brave-alert-lib/blob/main/.eslintrc.json

However, I know that @rhoberman was looking into sets of common ESLint config that we can use across all our projects. I'm not sure what ended up happening with that investigation. It looks like BeSafe is currently using this: https://github.com/bravetechnologycoop/besafe-app/blob/master/.eslintrc.json which is definitely different.

To make it run in Travis you can:
1. Add an `npm run lint` script to `package.json` like this: https://github.com/bravetechnologycoop/brave-alert-lib/blob/main/package.json#L11
2. Tell Travis to run it each time like this: https://github.com/bravetechnologycoop/brave-alert-lib/blob/main/.travis.yml#L13